patches/components-history-core-browser-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
+index 7ef197effead5ea14abca72fc94ae38a71d9a392..5c06bb1d68298e276de65d746fc410c5ddf95db4 100644
+--- a/components/history/core/browser/BUILD.gn
++++ b/components/history/core/browser/BUILD.gn
+@@ -156,6 +156,7 @@ static_library("browser") {
+       "android/visit_sql_handler.h",
+     ]
+   }
++  import("//brave/components/history/core/browser/sync/sources.gni") sources += brave_components_history_core_browser_sync_sources deps += brave_components_history_core_browser_sync_deps
```

Comment:
please put `sources.gni` at the same level where upstream `BUILD.gn` is located. In this case it should be `//brave/components/history/core/browser/sources.gni`.

and variables should omit `sync`:
`brave_components_history_core_browser_sync_sources` -> `brave_components_history_core_browser_sources`
`brave_components_history_core_browser_sync_deps` -> `brave_components_history_core_browser_deps`

Reply:
oh, indeed, I missed that, thanks, fixed at https://github.com/brave/brave-core/pull/16705/commits/d0d054c1a37a92a2e14d182add748599cc552bfe.